### ISSUE TITLE
GPL-nnn As a long read use I would like the binding kit barcode to be associated with the well

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-ui#514]
**User story**
Binding kit barcode should now be associated with the well. Pacbio have released a second kit so we could now potentially have 2 different kits per sample plate. More often than not we will be sequencing using only one kit per run but there may be occasions where we need to run two at the same time. Not urgent, will need to be addressed when more than ToL samples start going through TRACTION, but even then currently this library type is rare.

**Who are the primary contacts for this story**
Craig

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] When I go to the new run page I should not see binding kit barcode
- [ ] When I go to the wells modal on the new runs page I should see binding kit barcode

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
This was done as part of another story during the IsoSeq work.
